docker/cloudbuild.yaml

```diff
@@ -66,6 +72,19 @@ steps:
   wait_for: ['20.10.14']
   id: 'latest'
 
+- name: 'europe-docker.pkg.dev/cloud-builders/ga/v1/docker'
```

Comment:
Random improvement thought: can we avoid this manual tagging by tagging the latest tag during the actual build step? Looking at the other builders it should be possible.

docker build --tag=gcr.io/cloud-builders/docker .

Reply:
I will prefer to keep explicit step to tag latest for docker. Main reason for this is to avoid "implicitly" tagging multiple versions as latest. If we use a `--tag` in build step for version `20.10.14` then next time we move to newer version developer will need to remember to also remove `docker build --tag=gcr.io/cloud-builders/docker .` from step for version `20.10.14` and add same to new step. Having an explicit step for tagging latest, makes it easy to remember this and avoids mistake of not removing from old build step.